htdocs/include/cp_functions.php

```
@@ -120,6 +120,72 @@ function xoops_file_label($filename)
+ * @param string $path    Absolute path to the file.
+ * @param int    $perms   Permission bits (octal).
+ * @param string $context Short label used in the warning message
+ *                        (e.g. 'temp', 'temp guard').
+ *
+ * @return bool True on success, false on failure (warning already emitted).
+ */
+function xoops_chmod_quietly($path, $perms, $context = 'temp')
+{
+    $previousLevel = error_reporting(0);
+    try {
+        $ok = chmod($path, $perms);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $ok. Configured minimum length is 3. (undefined)

(ShortVariable)